test/puppetlabs/trapperkeeper/bootstrap_test.clj

```diff
@@ -154,21 +156,70 @@ puppetlabs.trapperkeeper.examples.bootstrapping.test-services/hello-world-servic
 puppetlabs.trapperkeeper.examples.bootstrapping.test-services/foo-test-service ; comment"
           service-maps      (->> bootstrap-config
                                  (StringReader.)
-                                 parse-bootstrap-config!
+                                 vector
+                                 parse-bootstrap-configs!
                                  (map service-map))]
       (is (= (count service-maps) 2))
       (is (contains? (first service-maps) :HelloWorldService))
       (is (contains? (second service-maps) :TestService)))))
 
+(deftest multiple-bootstrap-files
```

Comment:
I think it would be good to have a test where more than one service is present in an individual bootstrap file.  Maybe a combo test that does multiple files in one directory where each file has more than one service present in the bootstrap file.